spec/spec_helper.rb

```diff
@@ -1,3 +1,12 @@
+require 'simplecov'
+require 'coveralls'
```

Comment:
what's the point of "require: false" in the Gemfile when you just require it here?